src/components/common/ConnectWallet/MPCLogin.tsx

```diff
@@ -8,18 +8,41 @@ import InfoIcon from '@/public/images/notifications/info.svg'
 
 import css from './styles.module.css'
 import useWallet from '@/hooks/wallets/useWallet'
-import { useCurrentChain } from '@/hooks/useChains'
-import chains from '@/config/chains'
+import useChains, { useCurrentChain } from '@/hooks/useChains'
+import { isSocialWalletEnabled } from '@/hooks/wallets/wallets'
+import { ONBOARD_MPC_MODULE_LABEL } from '@/services/mpc/module'
+import { CGW_NAMES } from '@/hooks/wallets/consts'
+import { type ChainInfo } from '@safe-global/safe-gateway-typescript-sdk'
 
-const MPCLogin = ({ onLogin }: { onLogin?: () => void }) => {
+export const _getSupportedChains = (chains: ChainInfo[]) => {
+  return chains.reduce((result: string[], currentChain) => {
+    if (CGW_NAMES.SOCIAL_LOGIN && !currentChain.disabledWallets.includes(CGW_NAMES.SOCIAL_LOGIN)) {
+      result.push(currentChain.chainName)
+    }
+    return result
+  }, [])
+}
```

Comment:
Nit: I feel like this is a bit simpler to read
```suggestion
 const filteredChains = chains.filter((chain) => CGW_NAMES.SOCIAL_LOGIN && !currentChain.disabledWallets.includes(CGW_NAMES.SOCIAL_LOGIN))
 return filteredChains.map((chainConfig) => chainConfig.chainName)
```